docker-compose.dev.yml

```diff
@@ -155,6 +155,30 @@ services:
     env_file:
       - aleph.env
 
+  e2e:
+    build:
+      context: e2e
+    environment:
+      BASE_URL: "http://ui:8080"
+    volumes:
+      - "./e2e:/e2e"
```

Comment:
Maybe this could go into `docker-compose.e2e.yml` too, now that we have a separate config file?